cli_tools/daisycommon/daisy_utils.go

```diff
@@ -50,37 +50,53 @@ Loop:
 	return w, nil
 }
 
-// WorkflowAttributes holds common attributes that are required to instantiate a daisy workflow.
-type WorkflowAttributes struct {
-	Project, Zone, GCSPath, OAuth, Timeout, ComputeEndpoint, WorkflowDirectory string
-	DisableGCSLogs, DisableCloudLogs, DisableStdoutLogs, NoExternalIP          bool
+// EnvironmentSettings controls the resources that are used during import.
+type EnvironmentSettings struct {
+	// Location of workflows
+	WorkflowDirectory string
+
+	// Fields from daisy.Workflow
+	Project, Zone, GCSPath, OAuth, Timeout, ComputeEndpoint string
+	DisableGCSLogs, DisableCloudLogs, DisableStdoutLogs     bool
+
+	// Worker instance customizations
+	Network, Subnet       string
+	ComputeServiceAccount string
+	NoExternalIP          bool
+}
+
+// SetWorkerCustomizations sets variables that are used when creating worker instances.
+func SetWorkerCustomizations(wf *daisy.Workflow, env EnvironmentSettings) {
```

Comment:
Can this be a method on EnvironmentSettings?

Reply:
I was worried it'd be ambiguous which struct is being updated, since setters (often) update the receiving object. What do you think?

Reply:
Maybe rename the method at the same time? Like `UpdateWorkflow` or similar?

Reply:
sgtm. Changed to "Apply" instead of "Set".